### PR TITLE
Start variable numbering from 1 for app developer and requester secrets

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java [hunk 1]
```diff
@@ -52,13 +52,13 @@ public AppComputeSecretController(AuthorizationService authorizationService,
...
     // region App developer endpoints
-    @PostMapping("/apps/{appAddress}/secrets/0")
+    @PostMapping("/apps/{appAddress}/secrets/1")
     public ResponseEntity<ApiResponseBody<String, List<String>>> addAppDeveloperAppComputeSecret(@RequestHeader("Authorization") String authorization,
                                                                                @PathVariable String appAddress,
 //                                                                      @PathVariable String secretIndex,    // FIXME: enable once functioning has been validated
                                                                                @RequestBody String secretValue) {
         appAddress = appAddress.toLowerCase();
-        String secretIndex = "0";   // FIXME: remove once functioning has been validated.
+        String secretIndex = "1";   // FIXME: remove once functioning has been validated.
```
Comment (mcornaton) [inline src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java:61]: We could remove the `// FIXME` as we decided to go with a single index for a while.

**Review by jeremyjams — COMMENTED** [review]
Are you waiting for another PR to add alias?

File: src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java (hunk 1 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java]: done in ec9873f
I'm currently asking myself if we shouldn't remove the `secretIndex` notion and if it has a real meaning to have a `/apps/{appAddress}/secrets/1` endpoint.

**Comment by jbern0rd** [comment]
> Are you waiting for another PR to add alias?

Not at all, I forgot to commit palaemon template, now done in 4636c19

File: src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java (hunk 1 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java]: After giving some thoughts to it, I would prefer to handle the full removal in a next PR and to be able to push this fix as quickly as possible for Pierre.

Comment (mcornaton) [reply src/main/java/com/iexec/sms/secret/compute/AppComputeSecretController.java]: Sure, let's merge! :rocket:

**Pull request merged**